### ISSUE TITLE
GAP_PKGS_TO_BUILD is case sensitive, which causes breakage

### ISSUE THREAD

**Issue by fingolfin** [gap-actions/setup-gap#43]
Had to do this recently:
```
      - uses: gap-actions/setup-gap@v2
        with:
          # WORKAROUND: we have both zeromqinterface and ZeroMQInterface
          # in the list because GAP_PKGS_TO_BUILD is dumb and tries to
          # match packages based on directory names, and here the case
          # changed ...
          GAP_PKGS_TO_BUILD: "io profiling crypting json uuid zeromqinterface ZeroMQInterface"
          GAPBRANCH: ${{ matrix.gap-branch }}
```
Reason is that we pass the given names directly to `BuildPackage.sh`, which in turn uses them to match directory names. But these changed (and can change again).

Better would be to use e.g. `PackageManager` to deal with this cleanly...

**Comment by ChrisJefferson** [comment]
I was hitting this and made a PR before I noticed this issue -- #44

**Comment by ChrisJefferson** [comment]
fixed in #44
